client/packages/system/src/Patient/Insurance/InsurancePolicySelect.tsx

```diff
@@ -60,6 +61,9 @@ const getDefaultValue = (policyType: string, t: TypedTFunction<LocaleKey>) => {
         value: InsurancePolicyNodeType.Business,
       };
     default:
-      return { label: '', value: '' as InsurancePolicyNodeType };
+      return {
```

Comment:
Changed to default to Personal Policy.
I think this is better than having no default and an extra field to fill out...

Reply:
The Autocomplete takes  a `clearable` prop, so probably good to set `clearable={false}` to get rid of the X icon if we're not going to allow it.